common/scripts/copyReferenceFilesToStaging.js

```diff
@@ -0,0 +1,57 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+/*
+1. Creates a staging-directory for the docs build
+2. Copies the folders from itwinjs-core/docs and itwinjs-core/generated-docs.
+3. Provides the ability for users to give an absolute path
+4. Provides the ability to modify the default path (itwinjs-core/staging-docs) through CLI arguments
+Usage: node copyReferenceFilesToStaging.js itwinjs-core/core/test-staging-directory.
+*/
+const path = require('path');
+const process = require('process');
+let fse;
+try {
+  fse = require("fs-extra");
+} catch (err) {
+  console.log("Please check that fs-extra is installed");
+  return console.error(err);
+}
+
+const basePath = path.resolve(__dirname, "..", "..", "..");
+let dest;
+
+if (process.argv[2]) {
+  dest = process.argv[2];
+} else {
+  dest = "itwinjs-core/staging-docs";
```

Comment:
You assume the root repo directory is called "itwinjs-core" (not true on my machine), and your `basePath` assumes this script won't be relocated?
Considered using `git rev-parse` instead?

Reply:
I'm curious - under what circumstances would you `git rev-parse --show-toplevel` produce multiple lines of output?

Reply:
The output from `git rev-parse --show-toplevel` seems to be ending with `"\n"`. That was causing problems with the rest of the code so I got rid of that.